README.md

```diff
@@ -29,6 +29,12 @@ Or you can get your app name and version number such as `my-project/1.0` with
 
     \Tremby\LaravelGitVersion\GitVersionHelper::getNameAndVersion()
 
+Also, you can get the latest commit hash by calling this method:
+
+    \Tremby\LaravelGitVersion\GitVersionHelper::getHash(8)
+    
+`getHash()` method gets an integer which is the length of commit hash. If it's null, `getHash()` returns full width.
```

Comment:
- Please wrap this at 80 columns (or at the new sentence).
- "gets an integer" would be better as "accepts an integer", otherwise it could look like you're describing what it's returning.
- Replace start of second sentence with "If it's `null` or absent".